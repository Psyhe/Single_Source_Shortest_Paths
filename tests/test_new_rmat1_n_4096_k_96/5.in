4096 215 257
215 514 153
215 2066 33
216 6 38
216 776 15
216 2082 108
216 3412 245
216 1391 10
216 28 184
216 2848 161
216 1538 116
216 184 81
216 192 58
216 289 89
216 2222 132
216 1025 96
216 2205 50
216 1416 218
216 130 222
217 2064 172
217 3608 170
217 832 51
217 194 146
217 38 177
217 3972 122
218 1792 85
218 4 228
218 1035 112
218 56 32
218 2108 196
218 8 146
218 1543 58
218 532 7
218 256 213
219 3372 35
219 130 45
219 664 126
219 0 18
220 1554 156
220 160 107
220 0 248
220 1125 148
220 0 225
220 1536 148
221 1360 34
222 131 173
223 34 17
224 3090 41
224 1163 238
224 1553 19
224 2327 165
224 1088 15
224 32 50
224 537 156
224 1282 158
224 448 13
224 2328 214
224 2338 127
224 2386 152
224 1 120
224 11 218
224 787 12
224 51 215
224 26 195
224 2464 112
224 32 71
224 76 18
224 1576 248
224 1025 145
224 0 7
224 3144 55
224 33 236
224 864 110
224 3628 97
224 1552 65
224 0 32
224 64 232
224 69 15
224 24 89
224 257 105
224 1094 68
224 14 164
224 1024 244
224 2048 4
224 1041 41
224 2 162
224 2048 175
224 1162 194
224 24 126
224 784 216
224 330 6
224 1 73
224 3072 19
224 2083 178
224 2828 131
224 2416 134
224 2330 96
224 272 24
224 2374 75
224 416 234
224 2055 5
224 260 156
224 8 159
224 1154 98
224 612 191
224 101 244
224 977 82
224 8 143
224 1281 200
224 516 10
224 2145 215
224 516 67
224 3209 155
224 816 95
224 1376 130
224 32 17
224 8 82
224 144 51
224 2208 111
224 1542 197
225 3152 207
225 2070 96
225 2308 66
225 1545 170
225 1076 54
225 39 14
225 40 247
225 1028 111
225 2050 240
225 209 112
225 2308 79
225 3076 120
225 3632 55
225 800 52
225 64 94
225 0 227
225 1824 42
225 52 210
225 2140 236
225 64 184
225 1632 19
225 772 64
225 770 128
225 270 240
225 1280 131
225 165 70
225 2444 225
225 528 103
225 2844 57
225 64 59
225 640 44
225 1024 22
225 1284 167
225 1040 242
225 2052 208
225 0 168
226 0 91
226 340 114
226 3217 35
226 0 63
226 1537 122
226 1028 249
226 1285 253
226 2176 82
226 2440 104
226 2124 112
226 132 22
226 7 31
226 2576 102
226 784 232
226 600 90
226 2308 31
226 256 235
226 3336 50
226 0 73
226 1030 4
226 872 67
226 1697 44
226 803 206
226 288 69
226 524 206
226 2836 45
226 258 112
226 1102 152
226 513 111
226 272 193
227 16 36
227 1208 86
227 3336 153
227 68 81
227 2306 252
227 2372 128
227 140 230
227 1353 50
227 2354 89
227 3104 35
228 1 118
228 1026 157
228 516 195
228 0 22
228 520 172
228 138 151
228 1537 207
228 2060 222
228 36 164
228 98 54
228 8 84
228 833 179
228 121 83
228 1026 74
228 520 121
228 2176 197
228 6 189
228 0 138
228 256 24
228 2048 90
228 2816 177
228 71 90
228 780 23
229 72 15
229 1354 87
229 120 29
229 8 20
230 142 33
230 0 127
230 2444 136
230 1665 54
230 161 10
230 132 209
230 8 23
230 1288 73
230 2569 185
231 513 138
231 2100 50
231 4 48
231 2048 113
231 3088 138
231 3328 24
231 80 69
232 528 110
232 986 248
232 512 138
232 1352 35
232 1056 136
232 3125 201
232 320 34
232 613 201
232 264 216
232 1034 230
232 0 54
232 581 192
232 288 233
232 0 197
232 516 192
232 66 232
232 2049 232
232 64 134
232 4 207
232 3596 227
232 1035 140
232 257 231
233 578 202
233 1092 143
233 528 132
233 164 87
233 4 72
233 2712 64
233 8 202
234 2176 63
234 512 147
234 3208 73
234 2304 27
235 2348 193
236 2080 212
236 641 238
236 2418 237
236 2368 120
236 2373 48
237 3728 159
237 1 119
240 518 168
240 2049 52
240 768 79
240 1412 35
240 88 202
240 2048 73
240 2057 190
240 1925 106
240 1040 178
240 2064 105
240 2560 188
240 132 129
240 2176 3
240 310 211
240 257 64
240 1112 82
240 32 89
240 2049 2
240 1282 254
240 594 81
240 2561 96
240 1041 217
240 712 162
240 1793 214
240 964 125
240 73 68
240 256 237
241 96 235
241 1057 129
242 2048 96
242 129 248
242 2336 131
242 1030 171
243 584 31
243 130 111
243 3595 187
244 320 6
244 1794 23
244 1048 43
244 2052 176
244 24 243
244 41 216
244 914 53
244 1 203
244 2625 206
244 1 126
245 321 42
245 1161 183
245 1154 196
245 96 105
245 64 212
246 1284 24
246 512 68
248 786 140
248 0 106
248 4 83
248 513 208
248 1 196
248 1904 42
248 3316 242
248 1027 52
248 528 202
248 136 166
248 2080 167
248 386 193
248 1936 156
248 2432 15
248 5 185
249 27 202
249 132 20
251 36 16
252 512 222
252 2688 62
252 33 55
252 16 201
252 33 47
252 2820 251
253 1266 93
256 1592 162
256 8 131
256 1280 128
256 740 243
256 3256 40
256 34 173
256 3594 196
256 0 30
256 769 48
256 38 135
256 3152 148
256 165 44
256 3353 30
256 257 168
256 722 96
256 712 242
256 2344 127
256 1568 174
256 1120 228
256 1 18
256 279 254
256 584 165
256 264 54
256 2048 199
256 264 217
256 1673 135
256 1104 20
256 2066 0
256 518 30
256 0 225
256 1120 232
256 8 62
256 2152 122
256 192 208
256 1793 138
256 48 55
256 432 179
256 2576 129
256 512 189
256 0 14
256 968 189
256 2052 1
256 2240 186
256 261 57
256 1024 177
256 100 192
256 548 61
256 521 11
256 68 42
256 1032 68
256 523 167
256 1025 162
256 3104 65
256 1077 165
256 712 230
256 192 82
256 2193 238
256 538 48
256 96 238
256 202 209
256 912 131
256 64 117
256 1346 102
256 1024 51
256 1169 159
256 512 10
256 44 81
256 768 246
256 1029 21
256 240 237
256 8 237
256 595 131
256 1152 183
256 2272 214
256 349 209
256 3078 165
256 1424 200
256 576 186
256 769 44
256 1794 215
256 132 92
256 32 209
256 99 225
256 112 56
256 2111 32
256 3137 241
256 1720 189
256 836 10
256 0 246
256 2064 243
256 2176 2
256 82 115
256 0 6
256 2963 118
256 93 10
256 47 149
256 2371 43
256 2 109
256 2688 234
256 530 65
256 523 159
256 270 221
256 2176 6
256 48 214
256 9 235
256 656 151
256 21 111
256 521 60
256 42 183
256 192 127
256 1536 51
256 594 217
256 73 114
256 1152 224
256 35 90
256 1058 188
256 2916 146
256 384 191
256 3076 56
256 1 132
256 2820 251
256 130 111
256 772 62
256 0 177
256 516 204
256 389 108
256 136 12
256 522 182
256 1112 24
256 512 112
256 148 185
256 18 88
256 64 32
256 2 93
256 0 234
256 2576 109
256 3117 8
256 17 80
256 128 179
256 3972 119
256 1171 234
256 2112 91
256 608 70
256 1678 118
256 548 17
256 16 129
256 818 12
256 168 229
256 48 149
256 34 251
256 2192 233
256 1552 117
256 2196 114
256 512 96
256 544 252
256 1776 69
256 2852 15
256 1408 21
256 2208 47
256 2824 171
256 417 103
256 72 68
256 2193 141
256 1632 50
256 2051 168
256 264 185
256 3406 176
256 812 124
256 3616 120
256 514 146
256 1041 174
256 784 117
256 1088 207
256 0 222
256 592 110
256 556 132
256 722 154
256 2356 195
256 56 240
256 2048 200
256 3080 224
256 264 2
256 89 81
256 402 19
256 228 24
256 36 17
256 2062 230
256 16 117
256 78 190
256 2049 212
256 3648 254
256 36 127
256 1296 32
256 2116 39
256 143 0
256 2241 89
256 281 98
256 2057 187
256 322 204
256 288 165
256 1601 217
256 4 5
256 532 124
256 1308 75
256 581 254
256 0 100
256 448 185
256 1041 162
256 4 249
256 1153 55
256 7 92
256 1795 6
256 1043 155
256 1088 8
256 1620 3
256 640 62
256 334 113
256 649 124
256 70 201
256 257 27
256 67 200
256 3084 64
256 48 68
256 2336 147
256 2146 23
256 128 131
256 1328 9
256 353 62
256 3136 67
256 2096 233
256 1248 219
256 1604 39
256 2065 225
256 2562 226
256 16 28
256 3076 73
256 1696 222
256 136 216
256 1025 98
256 2368 254
256 1028 209
256 1792 174
256 2116 247
256 2204 252
256 514 175
256 405 16
256 32 161
256 133 162
256 840 241
256 2052 238
256 2064 248
256 3081 234
256 1047 48
256 2080 163
256 2572 170
256 705 97
256 2307 116
256 640 50
256 3105 230
256 2088 110
256 34 211
256 3201 235
256 260 75
256 74 204
256 64 0
256 0 68
256 1060 233
256 2405 61
256 512 3
256 3601 10
256 73 119
256 3682 162
256 1537 94
256 73 253
256 32 198
256 2096 34
256 1066 164
256 68 72
256 144 67
256 5 42
256 1036 1
256 2052 59
256 898 230
256 2061 68
256 264 255
256 2724 91
256 544 29
256 3073 58
256 52 132
256 417 177
256 548 18
256 1795 116
256 3076 175
256 39 222
256 1040 130
256 1316 186
256 22 2
256 144 75
256 32 19
256 16 24
256 1089 185
256 2176 52
256 1857 246
256 32 76
256 275 184
256 821 77
256 1049 172
256 832 159
256 0 113
256 276 33
256 1280 81
256 1068 198
256 348 48
256 47 69
256 2304 30
256 4 75
256 964 9
256 2172 74
256 0 117
256 1037 247
256 131 145
256 272 134
256 1040 248
256 560 146
256 1057 147
256 18 81
256 3718 191
256 3 175
256 277 214
256 80 184
256 306 123
256 10 167
256 16 199
256 1664 213
256 13 115
256 453 13
256 1288 9
256 1034 114
256 43 38
256 1 162
256 1601 108
256 512 32
256 160 249
256 132 244
256 2132 41
256 2 147
256 1544 5
256 5 124
256 3440 121
256 2569 72
256 48 252
256 10 94
256 1536 8
256 544 111
256 522 106
256 143 86
256 96 53
256 3093 70
256 536 85
256 64 224
256 646 148
256 32 231
256 2184 72
256 2076 84
256 768 85
256 1041 220
256 3840 6
256 17 138
256 838 78
256 745 48
256 3248 65
256 304 8
256 88 220
256 80 38
256 2053 240
256 7 178
256 226 235
256 2308 222
256 0 109
256 528 228
256 1568 225
256 656 116
256 1024 138
256 2208 101
256 1058 19
256 1088 204
256 3200 57
256 2 241
256 962 175
256 2240 204
256 27 245
256 152 132
256 3105 251
256 3888 108
256 522 171
256 17 12
256 560 142
256 2048 10
256 72 179
256 3620 53
256 564 23
256 772 55
256 144 165
256 2 58
256 296 102
256 2681 114
256 284 226
256 1680 19
256 2060 240
256 1601 157
256 112 13
256 480 8
256 2117 103
256 1651 33
256 656 47
256 2696 24
256 3076 29
256 18 4
256 2051 206
256 1609 238
256 144 145
256 3179 184
256 196 172
256 1334 220
256 2062 154
256 32 170
256 2081 22
256 64 139
256 3104 254
256 0 207
256 2392 235
256 0 20
256 1792 187
256 2197 249
256 1056 229
256 2582 253
256 2576 192
256 2560 197
256 5 100
256 208 162
256 2116 20
256 8 48
256 26 148
256 32 97
256 2613 131
256 528 90
256 4 173
256 1095 135
256 604 248
256 2328 177
256 164 110
256 69 223
256 2098 199
256 52 137
256 648 144
256 1156 253
256 2176 53
256 144 23
256 34 147
256 416 40
256 1058 125
256 0 191
256 1032 1
256 645 224
256 0 4
256 664 169
256 2 160
256 713 8
256 49 193
256 522 143
256 4 153
256 1028 182
256 2050 213
256 78 49
256 98 246
256 1801 103
256 2560 128
256 28 15
256 2056 250
256 1 126
256 2182 92
256 0 142
256 3717 202
256 786 222
256 1296 54
256 32 28
256 2180 180
256 261 161
256 1234 91
256 1200 106
256 850 235
256 1588 28
256 582 233
256 546 172
256 2048 96
256 1040 143
256 264 248
256 1062 12
256 34 78
256 3593 199
256 22 44
256 2180 184
256 0 122
256 577 133
256 705 21
256 2113 64
256 84 222
256 1040 66
256 2888 7
256 2064 95
256 321 58
256 2084 212
256 272 9
256 2566 13
256 720 228
256 2050 254
256 137 186
256 3252 20
256 1040 70
256 72 5
256 3586 244
256 392 197
256 2601 251
256 2059 210
256 1 248
256 1233 65
256 64 91
256 97 39
256 1544 10
256 1632 15
256 678 6
256 0 240
256 150 184
256 6 31
256 292 104
256 128 238
256 48 70
256 529 127
256 65 103
256 104 246
256 800 246
256 1420 139
256 2048 88
256 776 254
256 16 111
256 2048 35
256 74 63
256 2605 250
256 2369 158
256 16 192
256 4 129
256 1128 92
256 80 47
256 3209 71
256 514 205
256 136 6
256 16 62
256 32 191
256 176 23
256 160 55
256 1232 98
256 2752 197
256 0 94
256 74 177
256 1120 235
256 578 7
256 1024 70
256 904 126
256 72 107
256 212 5
256 9 26
256 840 7
256 2188 133
256 834 190
256 3200 70
256 196 120
256 2560 194
256 1034 144
256 3076 221
256 1152 149
256 2049 84
256 4 247
256 1026 120
256 1544 59
256 212 225
256 2593 65
256 3072 76
256 2 124
256 2050 230
256 81 230
256 34 168
256 1124 175
256 0 78
256 2577 44
256 2 234
256 2178 41
256 2134 92
256 1545 181
256 74 108
256 152 125
256 85 192
256 1 90
256 1030 86
256 16 50
256 1292 166
256 218 213
256 523 44
256 1300 220
256 257 139
256 260 36
256 946 94
256 13 47
256 3848 224
256 836 143
256 1749 231
256 9 67
256 1408 48
256 189 145
256 1 94
256 2321 166
256 71 221
256 2112 73
256 267 243
256 34 18
256 129 39
256 696 86
256 289 112
256 2050 27
256 2400 32
256 672 118
256 669 197
256 525 151
256 1706 30
256 803 93
256 1 74
256 128 141
256 3244 200
256 1122 143
256 9 128
256 785 103
256 0 253
256 3338 106
256 384 3
256 3600 77
256 2112 0
256 1920 180
256 529 221
256 545 165
256 3589 132
256 48 111
256 2097 174
256 1101 209
256 640 109
256 8 175
256 1 135
256 1602 223
256 4 219
256 387 118
256 512 5
256 3072 101
256 786 44
256 3600 191
256 1376 20
256 577 248
256 1216 86
256 128 235
256 19 76
256 3216 124
256 1058 252
256 1154 94
256 273 132
256 16 75
256 48 148
256 684 92
256 3089 248
256 274 251
256 615 223
256 0 54
256 12 64
256 1038 5
256 0 1
256 128 186
256 540 130
256 32 74
256 34 120
256 2308 61
256 260 252
256 1793 48
256 449 122
256 1570 191
256 32 131
256 1281 251
256 81 133
256 569 72
256 646 239
256 896 93
256 362 159
256 3913 197
256 3344 53
256 2054 95
256 73 219
256 3113 118
256 736 242
256 1102 79
256 261 81
256 2088 108
256 514 206
256 1044 155
256 0 119
256 1040 116
256 1281 178
256 566 209
256 2176 225
256 130 163
256 2560 28
256 566 156
256 1036 24
256 1592 231
256 2050 7
256 0 42
256 1060 207
256 610 105
256 1040 173
256 2153 245
256 128 150
256 0 99
256 105 3
256 549 237
256 1024 209
256 1064 92
256 2580 123
256 1320 165
256 1025 247
256 269 89
256 2049 166
256 186 212
256 37 172
256 2128 85
256 0 237
256 8 208
256 202 50
256 107 163
256 16 128
256 258 145
256 530 28
256 1312 174
256 2396 144
256 770 133
257 2178 111
257 256 168
257 86 30
257 12 138
257 64 112
257 2936 55
257 1760 250
257 17 189
257 2 129
257 2054 116
257 8 54
257 0 140
257 1281 146
257 321 76
257 3584 183
257 2066 53
257 1386 54
257 2560 134
257 640 130
257 560 43
257 1312 109
257 2266 204
257 0 18
257 1042 37
257 72 112
257 2048 77
257 2112 172
257 650 54
257 1 91
257 59 21
257 24 161
257 512 234
257 71 194
257 2048 114
257 800 8
257 260 147
257 0 169
257 1344 93
257 200 151
257 2496 150
257 512 108
257 280 108
257 1306 130
257 2976 39
257 3141 77
257 580 136
257 548 90
257 6 49
257 557 99
257 10 64
257 6 159
257 1024 47
257 1252 91
257 1024 214
257 1058 229
257 2072 78
257 522 223
257 68 2
257 650 87
257 1290 233
257 2048 167
257 256 27
257 152 212
257 128 86
257 260 176
257 1 100
257 128 143
257 3089 209
257 3104 86
257 6 98
257 80 159
257 296 219
257 320 134
257 1034 12
257 224 105
257 2626 97
257 528 113
257 546 130
257 112 110
257 3472 203
257 288 98
257 536 64
257 100 185
257 2176 60
257 46 246
257 545 53
257 1090 229
257 2 248
257 68 39
257 1091 113
257 580 247
257 1028 140
257 129 188
257 8 19
257 289 179
257 864 144
257 194 215
257 2056 212
257 1063 90
257 0 109
257 1153 117
257 2816 38
257 32 239
257 1604 231
257 2818 141
257 2562 202
257 1084 89
257 3160 178
257 1042 136
257 1088 212
257 133 61
257 5 10
257 21 88
257 60 201
257 232 231
257 2568 1
257 1025 54
257 6 209
257 4 112
257 3654 186
257 512 8
257 72 244
257 128 152
257 3852 122
257 1568 111
257 341 77
257 43 99
257 647 213
257 96 244
257 1280 183
257 176 201
257 32 48
257 1282 180
257 3621 144
257 550 101
257 452 50
257 2696 154
257 2180 74
257 2344 23
257 2049 132
257 240 64
257 168 137
257 22 206
257 3456 94
257 1281 136
257 3085 179
257 1072 230
257 524 128
257 2112 227
257 400 68
257 18 124
257 72 224
257 765 60
257 389 41
257 23 17
257 2077 208
257 2 50
257 2586 32
257 771 95
257 2 107
257 1024 114
257 2584 206
257 1072 161
257 2077 135
257 516 174
257 10 82
257 3476 12
257 5 35
257 2112 162
257 64 107
257 1056 197
257 3 233
257 2304 226
257 136 254
257 768 25
257 539 56
257 418 235
257 1161 105
257 546 209
257 2852 9
257 1024 159
257 0 29
257 130 207
257 2053 111
257 40 213
257 156 172
257 24 192
257 129 96
257 3072 11
257 280 25
257 96 42
257 136 226
257 386 187
257 2092 21
257 2372 196
257 25 155
257 644 95
257 2100 209
257 2697 254
257 2211 225
257 36 133
257 264 41
257 2324 135
257 624 63
257 136 234
257 33 55
257 2064 75
257 517 170
257 1297 109
257 523 68
257 1 224
257 116 204
257 3368 45
257 8 110
257 12 178
257 1312 153
257 145 170
257 337 100
257 144 28
257 1362 57
257 256 139
257 3092 208
257 1104 129
257 1682 229
257 1152 227
257 2627 152
257 384 60
257 48 111
257 2056 149
257 72 26
257 1344 157
257 1040 14
257 135 218
257 136 15
257 1160 98
257 30 50
257 737 46
257 2145 148
257 3141 247
257 5 154
257 40 124
257 138 12
257 808 75
257 1024 160
257 8 212
257 65 104
257 3114 14
257 1376 134
257 8 82
257 2 104
257 1152 162
257 1126 230
257 704 216
257 1847 178
257 1056 56
257 404 198
257 577 126
257 652 164
257 2624 224
257 720 111
257 1680 179
