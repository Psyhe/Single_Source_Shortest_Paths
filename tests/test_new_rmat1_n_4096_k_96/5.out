1000000000000000000
15
19
43
25
52
149
100
272
4
13
9
18
29
7
40
33
26
50
1
48
38
19
36
1000000000000000000
31
111
19
48
71
64
166
1000000000000000000
21
119
8
1000000000000000000
54
1000000000000000000
161
179
1
2
